### PR TITLE
GCP Logging Monitoring Updates

### PR DESCRIPTION
**Pull request by prat98**

### CODE REVIEW

**Review by itdove — APPROVED** [review]

File: stories/topics/con-gcp-customize-monitoring-logging.adoc [hunk 1]
```diff
@@ -3,9 +3,9 @@
 = Customizing monitoring and logging
 
 Metrics are provided by link:https://docs.ansible.com/automation-controller/latest/html/administration/metrics.html[Ansible], link:https://github.com/containers/prometheus-podman-exporter[Podman] and link:https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent[Google Ops Agent].
-Ansible metrics is only installed on {HubName}, the Google Ops Agent and Podman are also installed on {ControllerName}s.
+Ansible metrics is only installed on {HubName}, while the Google Ops Agent and Podman are installed on both {ControllerName} and {HubName} VM instances.
```
Comment (ianf77) [inline stories/topics/con-gcp-customize-monitoring-logging.adoc:6]: The Google Ops Agent and Podman are installed on {ControllerName} and {HubName} VM instances, however Ansible metrics are only installed on {HubName} instances.

File: stories/topics/proc-gcp-set-monitoring-logging-yml.adoc [hunk 2]
```diff
@@ -11,11 +11,11 @@ If you do not require a parameter, remove that parameter from the configuration
 * `cloud_credentials_path` is the absolute path toward your credentials. 
 This must be an absolute path.
 * `deployment_name` is the name of the deployment.
-* `components` is an array containing the type of component where you want to do the setup. 
+* `components` (Optional) is an array containing the type of component where you want to do the setup.
```
Comment (ianf77) [inline stories/topics/proc-gcp-set-monitoring-logging-yml.adoc:14]: Optional: components (in bacticks) is an array containing...

'the type of component where you want to do the setup' doesn't make much sense. 'the type of component on which you want to carry out the setup' ?

File: stories/topics/proc-gcp-set-monitoring-logging-yml.adoc [hunk 3]
```diff
@@ -11,11 +11,11 @@ If you do not require a parameter, remove that parameter from the configuration
 * `cloud_credentials_path` is the absolute path toward your credentials. 
 This must be an absolute path.
 * `deployment_name` is the name of the deployment.
-* `components` is an array containing the type of component where you want to do the setup. 
+* `components` (Optional) is an array containing the type of component where you want to do the setup. 
 The default is [ “controller”, “hub” ] which means that the logging monitoring will be enabled on both {controllerName} and {HubName}.
-* `monitoring_enabled` is set to `true` to enable the monitoring, `false` otherwise. Default = `false`.
-* `logging_enabled` is set to `true` to enable the logging, `false` otherwise. Default = `false`.
-* `default_collector_interval` is the periodicity at which the monitoring data must be send to the Google Cloud. 
+* `monitoring_enabled` (Optional) is set to `true` to enable the monitoring, `false` otherwise. Default = `false`.
```
Comment (ianf77) [inline stories/topics/proc-gcp-set-monitoring-logging-yml.adoc:16]: Optional: monitoring_enabled ....

File: stories/topics/proc-gcp-set-monitoring-logging-yml.adoc [hunk 4]
```diff
@@ -11,11 +11,11 @@ If you do not require a parameter, remove that parameter from the configuration
...
+* `logging_enabled` (Optional) is set to `true` to enable the logging, `false` otherwise. Default = `false`.
```
Comment (ianf77) [inline stories/topics/proc-gcp-set-monitoring-logging-yml.adoc:17]: Optional: logging_enabled...

File: stories/topics/proc-gcp-set-monitoring-logging-yml.adoc [hunk 5]
```diff
@@ -11,11 +11,11 @@ If you do not require a parameter, remove that parameter from the configuration
...
+* `default_collector_interval` (Optional) is the periodicity at which the monitoring data must be send to the Google Cloud.
```
Comment (ianf77) [inline stories/topics/proc-gcp-set-monitoring-logging-yml.adoc:18]: Optional: default_collector_interval

periodicity -> frequency ?

File: stories/topics/ref-gcp-ansible-podman-config.adoc [hunk 6]
```diff
@@ -68,9 +68,7 @@ collectors:
    sendInterval: 59s
 ----
 
-where
-
-`collectors` is a configuration array with one item per collector, that is, awx and podman.
+where `collectors` is a configuration array with one item per collector, that is, awx and podman.
```
Comment (ianf77) [inline stories/topics/ref-gcp-ansible-podman-config.adoc:71]: awx (in backticks)

**Review by ianf77 — COMMENTED** [review]
Some comments

File: stories/topics/proc-gcp-set-monitoring-logging-yml.adoc (hunk 2 above)
Comment (prat98) [reply stories/topics/proc-gcp-set-monitoring-logging-yml.adoc]: I used this format for the optional fields to remain consistent with the rest of the docs. I feel either we make this change everywhere or we keep it like this

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**